### PR TITLE
Step/logistics get notified

### PR DESCRIPTION
**Pull request by queenieho**

### CODE REVIEW

File: src/cljs/apply/sections/logistics/get_notified.cljs [hunk 1]
```diff
@@ -17,14 +20,29 @@
...
+(reg-sub
+ :db
+ (fn [db _]
+   db))
+
 ;; views ========================================================================
 
 
+;; NOTE I'm assuming that the date used here is for the user's
+;; selected date. If not, we have to revise this header
```
Comment (allenjprice) [inline src/cljs/apply/sections/logistics/get_notified.cljs]: this is correct - would you mind removing this comment from the code?

File: src/cljs/apply/sections/logistics/get_notified.cljs [hunk 2]
```diff
@@ -17,14 +20,29 @@
   false)
 
 
+;; subs =========================================================================
+
+
+(reg-sub
```
Comment (allenjprice) [inline src/cljs/apply/sections/logistics/get_notified.cljs]: This subscription is defined in the `apply.events` namespace - we don't need it here.

File: src/cljs/apply/sections/logistics/get_notified.cljs [hunk 3]
```diff
@@ -17,14 +20,29 @@
...
+  (let [date (subscribe [:step/data])]
+    [:div
+     [:div.w-60-l.w-100
+      [:h1 "We'll send you an email notification before " (format/date-month-day @date) " to remind  you to apply."]
+      [:p "Thanks for your interest in Starcity! Hope to see you soon."]]
+     [:div.w-80-l.w-100
+      [:div.page-content
+       [:a {:href "/logout"}
+        [button/primary
+         {:on-click #()}
```
Comment (allenjprice) [inline src/cljs/apply/sections/logistics/get_notified.cljs]: What is meant to happen when this button is clicked?

Comment (queenieho) [reply src/cljs/apply/sections/logistics/get_notified.cljs]: I should not be lazy and set up a route-to event...

**Pull request merged**